packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -643,4 +647,134 @@ export class JobService {
 
     return true;
   }
+
+  public async getDetails(userId: number, jobId: number): Promise<JobDetailsDto> {
```

Comment:
I think we can use subgraph to get all the data related to escrow, oracles, balance, token, etc .
We will need to add function to the sdk that returns the same as [this](https://sdk.humanprotocol.org/human-protocol-sdk/js-ts/escrow/escrowclient/getescrows) but called getEscrow and we pass the address as parameter

Reply:
Moved in the separate issue [[Job Launcher] [SDK] Implement getEscrow method to SDK and use it in getDetails method#892](https://github.com/humanprotocol/human-protocol/issues/892)